### PR TITLE
Regression Tests

### PR DESCRIPTION
**Pull request by pkjha-aero**

### CODE REVIEW

**Review by rafmudaf — APPROVED** [review]
This generally all looks good to me. The only thing I would add is a brief description of each test. This could be done in a README in the Tests directory, for example.

**Comment by rafmudaf** [comment]
I'd also like to merge #69 first so that we get the information on what additional code was tested here.

**Comment by rafmudaf** [comment]
I just saw issue #23 calling for READMEs for the test problems, so here's an up-vote for that.

**Comment by ewquon** [comment]
> This generally all looks good to me. The only thing I would add is a brief description of each test. This could be done in a README in the Tests directory, for example.

This is a good idea.

File: Tests/CTestList.cmake [hunk 1]
```diff
@@ -38,6 +38,7 @@ function(add_test_r TEST_NAME TEST_EXE PLTFILE)
     set(TEST_EXE ${CMAKE_BINARY_DIR}/Exec/${TEST_EXE})
     set(FCOMPARE_TOLERANCE "-r 1e-10 --abs_tol 1.0e-12")
     set(FCOMPARE_FLAGS "-a ${FCOMPARE_TOLERANCE}")
+    # TODO: Check if we need MPI_COMMANDS for running FCOMPARE_EXE
```
Comment (ewquon) [inline Tests/CTestList.cmake:41]: Agree, I don't think this is necessary. @rafmudaf is it better to just leave it in?

Comment (rafmudaf) [reply Tests/CTestList.cmake:41]: Good point, lets remove that just for our own future clarity.

Comment (rafmudaf) [reply Tests/CTestList.cmake:41]: Oh it looks like this was merged before this comment was addressed. I'll create a new pull request for this.

**Pull request merged**